doc/api/prev_api_changes/api_changes_3.8.0/behaviour.rst

```diff
@@ -183,6 +183,10 @@ Following the deprecation of ``pkg_resources.declare_namespace`` in ``setuptools
 As a consequence using ``pip`` to install a version of Matplotlib >= 3.8 on top
 of a version of Matplotlib < 3.8 (e.g. via ``pip install --local`` or
 ``python -m venv --system-site-packages ...``) will fail because the old
```

Comment:
Maybe: 
```suggestion
As a consequence using ``pip`` to install a version of Matplotlib >= 3.8 on top
of a version of Matplotlib < 3.8 (e.g. installed using a system package manager 
like ``apt install ...``, or via ``pip install --local`` or 
``python -m venv --system-site-packages ...``) will fail because the old
```